.github/workflows/build-test-release-client-packages.yml

```diff
@@ -287,7 +290,7 @@ jobs:
           ls -1 -R .
           echo ::set-output name=package_filenames::$(find ./build_artifacts -name cvmfs-config-eessi*)
 
-      - uses: meeDamian/github-release@2.0
+      - uses: meeDamian/github-release@7ae19492500104f636b3fee4d8103af0fed36c8e # v2.0.3
```

Comment:
This has surely got to require some permissions

Reply:
The `release` and `release-latest` actions are only run in certain scenarios, so can't test this in a PR

Reply:
I assume `contents: write` should be sufficient for this, but can't find it in the docs, so let's just try it out.